.github/workflows/release-manual.yml

```diff
@@ -0,0 +1,78 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
+        description: 'The branch to perform the release on, defaults to `release-$releaseVersion`'
+        type: string
+        required: false
+        default: ''
+      releaseVersion:
+        description: 'The version to be build and released. If no releaseBranch specified, expecting `release-$releaseVersion` to already exist.'
+        type: string
+        required: true
+      nextDevelopmentVersion:
+        description: 'Next development version, e.g. 8.X.X-SNAPSHOT.'
+        type: string
+        required: true
+      isLatest:
+        description: 'Whether this is the latest release and the docker image should be tagged as camunda/zeebe:latest'
+        type: boolean
+        required: false
+        default: false
+      dryRun:
+        description: 'Whether to push release commits and artifacts to remote (Git commits & tags, Maven artifacts, Github Release), defaults to true.'
+        type: boolean
+        default: true
+
+concurrency:
+  # cannot use the inputs context here as on this level only the github context is accessible, see
+  # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency
+  group: ${{ github.event.inputs.releaseBranch != '' && github.event.inputs.releaseBranch || format('release-{0}', github.event.inputs.releaseVersion) }}
+  cancel-in-progress: true
+
+jobs:
+  run-release:
+    name: "Release ${{ inputs.releaseVersion }} from ${{ inputs.releaseBranch }}"
+    uses: ./.github/workflows/release.yml
+    secrets: inherit
+    with:
+      releaseBranch: ${{ inputs.releaseBranch }}
+      releaseVersion: ${{ inputs.releaseVersion }}
+      nextDevelopmentVersion: ${{ inputs.nextDevelopmentVersion }}
+      isLatest: ${{ inputs.isLatest }}
+      dryRun: ${{ inputs.dryRun }}
+  notify-if-failed:
+    name: Send slack notification on failure
```

Comment:
:thinking: Didn't you mentioned you wanted to remove it from the manual executed?

Reply:
I would keep it for non dry runs, so we have higher attention/transparency on failures of actual hot release builds, thus the condition in line 49:
```
if: ${{ failure() && inputs.dryRun == false }}
```
However thinking about it, the reasoning I brought up for dry runs that "whoever ran it will look for the outcome" would also apply in such a case ^^ Still for actual releases you might argue it's worth to make failures transparent in zeebe-ci 🤔 
wdyt?

Reply:
I'm fine in both cases, I was just a bit confused to see it since you brought it up as point to not have it :D

Reply:
Just to double-check:

What I changed is moving the slack notification job to each of the new invoking workflows to account for special conditions and messaging.

`failure in release-manual:`
if: it's a dry run => no slack notification (**remove noise of manual testing**)
else slack notification indicating an actual release failed => **this is urgent and needs attention**

`failure in release-main-dry-run.yml:`
=> slack notification indicating a test run failed (not to be confused with a real release) => **needs investigation by e.g. medic**

We can adjust that anytime